tests/jcond.s

```diff
@@ -0,0 +1,31 @@
+jmp #5                      ; keep pass/fail at fixed location
+print ^cond_jump_insts_ok   ; overall pass
+halt
+print ^cond_jump_insts_fail ; any fail
+halt
+sub $1 $2 #3                ; test all j insts that don't consider < case as match
```

Comment:
Another reason is because your comment write "all j insts", but there only have three j insts, it will confuse with all j operator.